crates/nu-cli/src/completions/file_completions.rs

```diff
@@ -146,6 +146,7 @@ pub fn file_path_completion(
                             || path.contains('"')
                             || path.contains(' ')
                             || path.contains('#')
+                            || path.contains('(')
```

Comment:
At first I thought we also need to include the closing `)` but the nu parser works in mysterious ways...
![image](https://user-images.githubusercontent.com/15833959/211396212-3e8ef9d6-aeab-4310-b38c-3825de79cfdc.png)

For good measure:

```suggestion
                            || path.contains('(')
                            || path.contains(')')
```